src/scripts/migrateTenantOrgData.js

```
@@ -0,0 +1,2438 @@
+			referencedIds.set(metaKey, (referencedIds.get(metaKey) || 0) + ids.length)
+
+			const groupKey = `${spec.service}|${spec.endPoint}`
+			if (!groups.has(groupKey)) {
+				groups.set(groupKey, {
+					service: spec.service,
+					endPoint: spec.endPoint,
+					oldIds: new Set(),
+				})
+			}
+			const group = groups.get(groupKey)
+			ids.forEach((id) => group.oldIds.add(id))
```

Biome (2.4.7)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)